test/DocumentFormat.OpenXml.Tests/Spreadsheet/CellValueTests.cs

```diff
@@ -74,6 +76,26 @@ public void CellDoubleTest(double num)
             Assert.Equal(num, result);
         }
 
+        [Fact]
+        public void CellDoubleCultureTest()
+        {
+            // Change current culture
+            CultureInfo culture, oldculture;
+            oldculture = Thread.CurrentThread.CurrentCulture;
+            culture = CultureInfo.CreateSpecificCulture("fr-FR");
```

Comment:
It would be good to have a few cultures to verify this: ie en-US, fr-FR, ru-RU, zh-CH, tr-TR, ar-EG, etc (get in different ways of handling commas, left-to-right, etc)

Reply:
Yes, I'll spend some more time on this to see what makes sense.